pipeline-scripts/release.groovy

```diff
@@ -130,6 +143,9 @@ def getArchPrivSuffix(arch, priv) {
     return suffix
 }
 
+def stageValidateRHCOS(from_release_tag) {
+}
+
```

Comment:
did you leave this in by accident? i don't think we need a separate stage for this.

Reply:
Right... will remove this